src/colorizer/types.ts

```diff
@@ -319,12 +319,16 @@ export enum LoadTroubleshooting {
     " Please check your network access.",
   CHECK_FILE_EXISTS = "Please check if the file exists and if you have access to it, or see the developer console for more details.",
   CHECK_FILE_OR_NETWORK = "This may be because of an unsupported format, missing files, or server and network issues. Please see the developer console for more details.",
+  CHECK_ZIP_FORMAT = "Make sure a 'collection.json' or 'manifest.json' file exists in the root directory.",
+  CHECK_ZIP_FORMAT_COLLECTION = "A 'collection.json' should exist in the root directory.",
+  CHECK_ZIP_FORMAT_MANIFEST = "A 'manifest.json' should exist in the root directory.",
```

Comment:
*TECHNICALLY* this file doesn't need to be in the root directory, it just has to be in the highest folder in the hierarchy that contains files (and not just another nested directory). I'm not sure if there's a good way to word this, so for now "root directory" seems fine.

Reply:
"base" maybe?